engine/src/main/scala/cromwell/engine/workflow/lifecycle/execution/EngineJobExecutionActor.scala

```diff
@@ -515,11 +515,11 @@ class EngineJobExecutionActor(replyTo: ActorRef,
 
     data.ejha match {
       case Some(ejha) =>
-        log.info("Trying to use another cache hit for job: {}", jobDescriptorKey)
+        workflowLogger.debug("Trying to use another cache hit for job: {}", jobDescriptorKey)
```

Comment:
is the stringification of the `jobDescriptorKey` okay?

Reply:
it's not *great* but it's surprisingly readable

Reply:
Example:
```
Trying to use another cache hit for job: BackendJobDescriptorKey_TaskCallNode_foo.bar:20:1
```

Reply:
I can has workflow id?

Reply:
Yeah, that's why I also switched it to workflowLogger

Reply:
sweet